### ISSUE TITLE
Library Code Not Working

### ISSUE THREAD

**Issue by ajdeshmane0007** [Secreto31126/whatsapp-api-google-app-script#1]
when adding in libraries erorr showing like this 
Unable to look up library. Check the ID and access permissions and try again.

**Comment by Secreto31126** [comment]
Feel free to retry! I forgot to make the code public, my bad 😄

**Comment by ajdeshmane0007** [comment]
ok working now 

can contact you for more info i have to do a project

plz share your whatsapp so i connect you

**Comment by Secreto31126** [comment]
I would rather keep the conversation here if you don't mind. Ask me anything you want. Also, the proyect has a [documentation](https://whatsappapijs.web.app/) if you ever feel a little bit lost with the classes, there are a few of them.

**Comment by ajdeshmane0007** [comment]
i am not expert in coding and i want to do a bot with this script you should work on it i will pay for it........

**Comment by Secreto31126** [comment]
Sorry, right now I'm not free to work in new proyects, I have some exams in a few weeks. However, I wouldn't mind listening what's your idea, and I could tell you if it's viable or not.

**Comment by ajdeshmane0007** [comment]
ok jst tell me one thing  is it work 24/7 without server or localhost through only sheet???

**Comment by Secreto31126** [comment]
Yes, it's posible to set up a server using the functions doGet and doPost, which will stay open 24/7. You can even create independent Google Script Proyects if you don't need the spreadsheet.

What's more, GAS has some cool crons (code executed on specific moments, such as a timer or incoming events in your calendar), and you can set up some really interesting things, such as send a WhatsApp reminder to all the participants of a Calendar event, or send a message when someone fills a Google Form (that's a little bit more tricky, but still pausible).

In conclusion, yeah, Google App Script stays active 24/7 and can do a lot buch of things.

**Comment by ajdeshmane0007** [comment]
Can we use other free api like venom, wa web js

**Comment by Secreto31126** [comment]
_You_ can do whatever you want. However, keep in mind those "API" aren't official and, actually, against WhatsApp terms of services. If they find you using those programs, they will more likely terminate your account. Plus, you would need a real server to host them, since GAS isn't a Node enviroment.
